api-docs/api.yaml

```diff
@@ -0,0 +1,86 @@
+swagger: "2.0"
+info:
+  version: 2.4.0
+  title: Koverse API
+  description: >
+        Koverse Web API Specification
+
+
+
+host: koverse.swagger.io:8000
```

Comment:
Does this need to be changed?

Reply:
Yes this should be changed to something that works but I don't know what